include/dsn/dist/distributed_lock_service.h

```diff
@@ -69,7 +69,7 @@ namespace dsn
             /*
              * initialization routine
              */
-            virtual error_code initialize() = 0;
+            virtual error_code initialize(const char* work_dir, const char* lock_root) = 0;
```

Comment:
Ok, I think this way may be proper:
Set lock_root in config by "distributed_lock_service_arguments":
```
[meta_server]
distributed_lock_service_type = meta_state_service_zookeeper
distributed_lock_service_arguments = /a/b/c
```
Change initialize() method as:
```
virtual error_code initialize(int argc, char** argv);
```
so as the meta_state_service, how do you think?